docker/entry.sh

```diff
@@ -29,6 +29,27 @@ map_dist() {
 	return 0
 }
 
+host_dist_matches_image_dist() {
+	if [ -z "$LB_DIST" ]; then
+		[ -f "$HOSTRELEASE" ] || die "You have to bind-mount /etc/os-release to the container's $HOSTRELEASE"
+		host_dist="$(lbdisttool.py -l --os-release $HOSTRELEASE | cut -d'.' -f1 )"
+	else
+		host_dist="$LB_DIST"
+	fi
```

Comment:
this is an exact re-implementation of `map_dist`. Use that function instead and do your `cut` ing on its result

Reply:
done